packages/open-truss/src/configuration/apply.tsx

```diff
@@ -38,3 +40,22 @@ export function applyConfiguration(
 
   return configurationFunction
 }
+
+interface RenderFromFile {
+  components?: COMPONENTS,
+  path: string
+}
+
+export async function RenderFromFile({ components = {}, path }: RenderFromFile): Promise<JSX.Element> {
+  let config
+  try {
+    config = await fs.readFile(path, 'utf-8')
+  } catch (err) {
+    throw err
```

Comment:
This isn't good. See https://github.com/open-truss/open-truss/pull/57#discussion_r1432205144, I'm not sure how to handle this yet. Maybe this can't be in the OT package? That would be sad though.